New Airport ~ Manchester Airport w/ new airlines

eliuuk opened the pull request:
Manchester Airport is the 3rd busiest airport in the UK, and here it is!
- All SIDS (ASMIM, DESIG, EKLAD, KUXEM, LISTO, MONTY, Pole Hill,  SONEX and SANBA)
- All STARS (DALEY, ROSUN, MIRSI and DAYNE)
- Some additional airlines:
   Brussels Airlines,
   Monarch,
   Jet2 and
   Flybe
- Some additional aircraft:
  Avro RJ-100
  Saab 2000,
  Saab 340, and the
  de Havilland Canada DHC-6 Twin Otter

Brussels was also edited to include the national airline.

Commit:
RJ1H

Dependency for the Brussels Airlines creation

Commit:
Brussels Airlines

Brussels Airlines creation, only for Brussels at the moment

Commit:
Update ebbr.json

Commit:
Start of Manchester Airport

Commit:
Long Lat Speech Marks removed

Commit:
Update egcc.json

Commit:
Rename egcc.json to egcc2.json

Commit:
Update airport.js

Commit:
Create eddt.json

Commit:
Rename egcc.json to egcc.txt

Commit:
Rename egcc2.json to egcc.json

Commit:
Update airport.js

Commit:
Rename assets/airports/egcc.txt to assets/info.txt

Commit:
Update egcc.json

Commit:
Update egcc.json

Commit:
Added Waypoints, a SID and a STAR.

STAR does not work, as aircraft do not spawn.

Commit:
Removed info.txt

Was temporary, no longer needed.

Commit:
Update egcc.json

Commit:
Added Waypoints

Added EKLAD, KUXEM and ASMIM along with D255P for the EKLAD, KUXEM and ASMIM SIDS.

Commit:
Finalised EGCC

This includes all the additions required for EGCC, however, the SAAB 340
(sf34) needs looking at before a ph is requested. All SIDS and STARS
added, alphabetically sorted too!

Commit:
Update eddt.json

Commit:
Update egcc.json

Commit:
Update egcc.json

Commit:
Update egcc.json

Commit:
Merge pull request #1 from zlsa/gh-pages

ok

Commit:
Removed QTR

erikquinn commented on assets/airports/egcc.json:
```diff
@@ -0,0 +1,582 @@
+  "name": "Manchester Airport",
+  "level": "medium",
+  "radio": {
+    "twr": "Manchester Delivery",
```
Are you certain this is called "Manchester Delivery"? The one who tells them cleared to land and cleared for takeoff?

eliuuk replied:
It's not Manchester Delivery, thanks for reminding me to change that! The charts say "TWR : MANCHESTER DELIVERY" whereas the actual tower is Manchester Tower, which is the landing and takeoff clearance.

erikquinn commented on assets/airports/egcc.json:
```diff
@@ -0,0 +1,582 @@
+		"_05L":		["N53d20m51.200", "W002d17m15.950"],
+		"_05R":		["N53d19m55.100", "W002d18m38.380"],
+		"_23L":		["N53d21m40.750", "W002d15m33.410"],
+		"_23R":		["N53d20m53.350", "W002d16m37.950"],
```
These two fixes are backwards. It looks like `"_23L"` and `"_23R"` should be switched.

Commit:
Changed Tower Name

Commit:
Update egcc.json

Commit:
Switched _23L and _23R around and set rwy length

Runways were reset, and 23L and 23R, thanks to erikquinn for noticing this!

eliuuk commented:
Again, thanks for seeing that - got mixed up again with those two!

eliuuk commented:
'aircraft taxi' seems to have an issue. Specifying a runway through 'aircraft taxi runway' only seems to work.

Error:
http://prntscr.com/b6e316

Commit:
Changed the winds

Attempt to fix the 'aircraft taxi' issue.

Commit:
Merge pull request #2 from zlsa/gh-pages

Merge

erikquinn commented:
interesting...

eliuuk commented:
05L and 05R SIDS do not work, 23L and 23R SIDS or SIDS for all runways do work.
-> ASMIM
-> DESIG
http://prntscr.com/b6enu2

Commit:
Test

Testing to see if switching the runways to 23s will help.

Commit:
Reverted Test

erikquinn commented:
Depends on resolution to #519.

erikquinn commented:
@indianbhaji if you refresh until the game chooses to use the 5's, then they will work and the 23's will not. Currently, aircraft aren't smart enough to file for SIDs that are valid from the runways currently in use, or smart enough to realize that they can't fly the SID if they do depart on that runway. The latter is causing a crash, which I am working on fixing now. The former will happen eventually, but not very soon.

eliuuk commented:
Alright, thanks, for the time being I shall seek to remove those SIDS and reinstate them when this issue is fixed.

Commit:
Removed the 05L and 05R SIDS

This temporarily resolves the issue #519. The SIDS, DESIG and ASMIM will be reinstated after the bug is fixed.

erikquinn commented:
@indianbhaji Noooo, leave them in there! I've just about got it fixed. haha

eliuuk commented:
aha, I'll put them back!

Commit:
Reinstated the 05 departures

I made a boo boo of removing them.

Commit:
EGCC is much more complicated than I thought

Commit:
Merge pull request #3 from zlsa/gh-pages

EGCC airport fix

eliuuk commented:
Airport finished. I would go further with a custom airspace, but the one that exists in real life seems to be quite confusing.

erikquinn commented:
Merged via d78102ecfdad7cfb1e125cfd4ca037118cca7075. Closing.
